Makefile

```diff
@@ -1,7 +1,7 @@
 PREFIX=/usr/local
 DESTDIR=
 version=$(shell git describe --tags 2> /dev/null || git rev-parse HEAD)
-GOFLAGS=-ldflags "-X github.com/pcelvng/task-tools.Version=${version} -X github.com/pcelvng/task-tools.BuildTimeUTC=`date -u '+%Y-%m-%d_%I:%M:%S%p'`"
+GOFLAGS=-ldflags "-s -w -X github.com/pcelvng/task-tools.Version=${version} -X github.com/pcelvng/task-tools.BuildTimeUTC=`date -u '+%Y-%m-%d_%I:%M:%S%p'`"
```

Comment:
What does the -s and -w options do?

Reply:
-s     Omit the symbol table and debug information.
-w.    Omit the DWARF symbol table.

It doesn't change stack traces, just removes some debugging info from the binary.